build/setup.mk

```diff
@@ -1,14 +1,16 @@
+GO_BUILD_FLAGS ?=
+
 # Ensure that go is installed. Note that this is independent of whether or not a server is being
 # built, since the build script itself uses go.
 ifeq ($(GO),)
     $(error "go is not available: see https://golang.org/doc/install")
 endif
 
 # Ensure that the build tools are compiled. Go's caching makes this quick.
-$(shell cd build/manifest && $(GO) build -o ../bin/manifest)
+$(shell cd build/manifest && $(GO) build -buildvcs=false $(GO_BUILD_FLAGS) -o ../bin/manifest)
```

Comment:
@mickmister, this comes from your changes, is it needed? what's the goal?

Reply:
This was originally to have the workflow runner be able to build the plugin in a docker container. This can be removed as that's not how we're running the tests.